core/jvm/src/main/scala/me/shadaj/scalapy/py/JepInterpreter.scala

```diff
@@ -209,21 +209,21 @@ trait JepPyValue extends PyValue {
     }
   }
 
-  def getTuple = getSeq
+  def getTuple: Seq[PyValue] = getSeq
 
   def getSeq: Seq[PyValue] = {
```

Comment:
The problem with the views is when data get moved from python to scala, in the `getSeq` method. One solution, as you point, can be to change the method to getIterable, but because if in the scala side, I think user code will have to be modified accordingly, so I just think is better to remain with Seq's and to remove views. Depending on the use case, one might want to use LazyLists, but doesn't exists in scala 2.12 and Streams doesn't exists in 2.13, and are just lazy but have the same memory footprint of others collections once fully read...
it's a bit tricky to cross compile when view's are used.